FirebaseAuth/Tests/Unit/RPCBaseTests.swift

```diff
@@ -99,7 +99,7 @@ class RPCBaseTests: XCTestCase {
         XCTFail("decodedRequest is not a dictionary")
       }
       // Dummy response to unblock await.
-      let _ = try self.rpcIssuer?.respond(withJSON: [:])
+      let _ = try self.rpcIssuer?.respond(withJSON: ["sessionInfo": "verification_id_123"])
```

Comment:
why was this change needed?

Reply:
Thanks, I don't believe this is needed anymore. It was when I had the property mapping to `sessionInfo` non-optional, but it remained optional and this placeholder value is no longer needed.